src/test/objectstore/bmap_allocator_replay_test.cc

```diff
@@ -115,16 +147,22 @@ int replay_and_check_for_duplicate(char* fname)
 	std::cerr << "error: allocate: " << s << std::endl;
 	return -1;
       }
+      tmp.clear();
       auto allocated = alloc->allocate(want, alloc_unit, 0, 0, &tmp);
       std::cout << "allocated TOTAL: " << allocated << std::endl;
-      interval_set<uint64_t> intervals;
+      for (auto& ee : tmp) {
```

Comment:
We must verify that `alloc->allocate` actually gives the same regions that were mentioned in log. Otherwise `intervals` is useless to deduce any allocation error.

Reply:
Agree but it complicates the handling hence not implemented for now